File: backend/WebAPI/WebAPI/Models/Meeting.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace WebAPI.Models
{
    public class Meeting
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public uint Id { get; set; }
        public uint AdvertId { get; set; }
        public uint VisitorId { get; set; }
        public DateTime Time { get; set; }
        public DateTime DateCreated { get; set; }
        public bool AgreedVisitor { get; set; }
        public bool AgreedOwner { get; set; }
        public bool Concluded { get; set; }
    }
}

File: frontend/app/src/main/java/com/example/skucise/Meeting.kt
package com.example.skucise

import java.time.LocalDateTime

data class Meeting(
    val id: Int,
    val advertId: Int,
    val otherUser: Int,
    val username: String,
    val title: String,
    var proposedTime: LocalDateTime,
    val dateCreated: LocalDateTime,
    var agreedVisitor: Boolean,
    var agreedOwner: Boolean,
    val concluded: Boolean,
    val owner: Boolean
)
